### ISSUE TITLE
[Accessibility]: Collapsible sections in code-intel index pages don't explain what they contain

### ISSUE THREAD

**Issue by Strum355** [sourcegraph/sourcegraph#35263]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33513

### Problem description

Upon expanding the below collapsible, the screen reader instantly starts reading the `ignite exec executor-f7ceeb6b-8973-4d54-98a3-ef86ed9f09a3...` section, with no indication as to this being the command that was run for this step.

The "log output" section is preceeded by a text element indicating as such, but without it being directly linked to being part of the following section as opposed to the previous.

![image](https://user-images.githubusercontent.com/18282288/167709419-95a62c71-6890-45c6-b354-b3de949409e3.png)

### Expected behavior

The user is informed of what is coming next as opposed to being hit with a shell command that is often badly pronounced by the screen reader.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
@gitstart-sourcegraph I think we can give this a `VisuallyHidden` label, like `"Log output"`.

The label can be `"Executed command:"`


@Strum355 Assigning to GitStart, LMK if you disagree with anything!
